Web/amazon-connect-ccp-web-calling-example/src/components/MyCustomCCP/ConferenceView/DisplayPanel/ControlBar/ScreenSharingSessionButton/ScreenSharingSessionButton.js

```diff
@@ -0,0 +1,46 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: MIT-0
+
+import contactManager from '../../../../../../services/ContactManager';
+import eventBus from '../../../../../../utils/EventBus';
+
+class ScreenSharingSessionButton extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  get screenShareButton() {
+    return this.querySelector('#screen-sharing-session-button');
+  }
+
+  connectedCallback() {
+    this.render();
+
+    eventBus.on('ScreenSharingStateChanged', () => {
+      this.render();
+    });
+  }
+
+  attachEvents() {
+    this.screenShareButton.addEventListener('click', async () => {
+      await contactManager.toggleScreenSharingSession();
+    });
+  }
+
+  render() {
+    this.innerHTML = `
+      <style>
+        #screen-sharing-session-button {
+          height: 50px;
+        }
+      </style>
+
+      <button id="screen-sharing-session-button">
```

Comment:
nit: this is not likely to happen, but looks like if we instantiate multiple screen share buttons on the same page, they all will share the same ID. Maybe pass the ID in as a constructor parameter?

Reply:
Our idea is to keep it as simple as possible. And we're sure that there will be only one screen share button. It's not a reusable component. So no need to handle this.